asset_bar_op.py

```diff
@@ -119,6 +119,86 @@ def modal_inside(self, context, event):
             if change:
                 context.region.tag_redraw()
 
+        # Check for tab shortcut keys directly in the modal function
+        if (
+            event.ctrl
```

Comment:
On MacOS CMD+T, CMD+1..9 are the shortcuts in Safari and Firefox to create a new tab and switch tabs etc. Not the CTRL really.